js/tabs/js/lumx.tabs_directive.js

```diff
@@ -75,6 +76,184 @@ angular.module('lumx.tabs', [])
             }
         };
 
+        function isPaginationActive() 
+        {
+            var tabsWidth = links.outerWidth();
+            var tabsVisibleWidth = links.parent('.tabs').outerWidth();
+
+            return tabsWidth > tabsVisibleWidth;
+        }
+
+        function getFirstHiddenLeftTab()
+        {
+            var linkContainer = links.parent('.tabs');
+            var leftBorderContainer = linkContainer.offset().left;
+            var tabTags = links.find('.tabs-link');
+
+            var firstTabHidden;
+
+            for (var i = 0; i < tabTags.length; i++) 
+            {
+                var leftTab = $(tabTags[i]).offset().left;
```

Comment:
It's better to use `angular.element`. We want to remove the jquery dependency as soon as possible. Using the angular way right now wil ease the task.